docs/source/channel_update_tutorial.rst

```diff
@@ -565,8 +527,7 @@ chaincode by approving the chaincode definition.
           `Adding an org to a channel tutorial <https://hyperledger-fabric.readthedocs.io/en/release-1.4/channel_update_tutorial.html>`__.
 
 Before we install a chaincode as Org3, we can use the ``./network.sh`` script to
-deploy the Basic chaincode on the channel. Open a new terminal outside the
-Org3CLI container and navigate to the ``test-network`` directory. You can then use
+deploy the Basic chaincode on the channel. Open a new terminal and navigate to the ``test-network`` directory. You can then use
 use the ``test-network`` script to deploy the Basic chaincode:
 
 .. code:: bash
```

Comment:
This comment needs to be updated to deploy the smart contract to channel1 if you keep the channel name change:
```
./network.sh deployCC -ccn basic -ccp ../asset-transfer-basic/chaincode-go/ -ccl go -c channel1
```

Reply:
Done